Dockerfile

```diff
@@ -152,6 +166,26 @@ RUN tar xf node-v${NODEJS_VERSION}-linux-x64.tar.xz && \
     rm -fr node-v${NODEJS_VERSION}-linux-x64 && \
     rm -f node-v${NODEJS_VERSION}-linux-x64.tar.xz
 
+# Install gradle
+RUN curl -Lo gradle.zip https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip
```

Comment:
Could we squash these two commands into one like with clojure? Would save us one extra layer